### ISSUE TITLE
[Integration-tests] are missing for Trexle

### ISSUE THREAD

**Issue by oyeb** [aviabird/gringotts#105]
Integration tests must test each request's life; for example, there should be a test for these:
* `authorize` -> `capture` -> `refund`
* `authorize` -> `void`

**Comment by hossamhossny** [comment]
Do you guys have any account specific problems with your tests? @jyotigautam @oyeb

**Comment by jyotigautam** [comment]
@hossamhossny Yes, that is an account specific problem since I am unable to get any response to write the tests.

**Comment by hossamhossny** [comment]
@jyotigautam You can just open a new account since it is free anyways! You need to add Trexle API keys as the username and password when adding "Trexle sandbox" payment gateway in Trexle dashboard. 

Technically it is the only payment gateway available to you in the dashboard since it is a free account and I can't see any troubles doing this. Any commercial Trexle account will enable +150 supported payment gateways to add their API keys, but again in a free environment you can only work with "Trexle sandbox" and your issued Trexle APIs.

Also using this secret sandbox test key is possible,,
https://github.com/activemerchant/active_merchant/blob/master/test/fixtures.yml#L1178-L1179
